Source/Scene/Camera.js

```diff
@@ -2984,16 +2985,19 @@ Camera.prototype.getPickRay = function (windowPosition, result) {
     result = new Ray();
   }
 
+  const canvas = this._scene.canvas;
   const frustum = this.frustum;
-  if (
+  if (canvas.clientWidth <= 0 || canvas.clientHeight <= 0) {
+    return undefined;
+  } else if (
     defined(frustum.aspectRatio) &&
     defined(frustum.fov) &&
     defined(frustum.near)
   ) {
     return getPickRayPerspective(this, windowPosition, result);
+  } else {
```

Comment:
We favor returning from a function at the top-most level we can, and enforce this using the [`no-else-return`](https://eslint.org/docs/rules/no-else-return) eslint rule. That means that this should be written to omit the `else` as, since we returned in the other two conditions, it's not necessary. You can also factor out the `else if` statement, so that the pattern becomes

````js
if (condition) {
   return undefined;
}

if (secondCondition) {
   return getPickRayPerspective(this, windowPosition, result);
}

return getPickRayOrthographic(this, windowPosition, result);
````

Reply:
Hi @ggetz. Thank you for the styling feedback. I just pushed two commits that should address this matter.